### PR TITLE
MS16958: deactivate Create App when desktop windows are closed

### PR DESCRIPTION
**Pull request by thoys**
Fixes https://highfidelity.fogbugz.com/f/cases/16958/Create-menu-does-not-close-when-its-windows-are-closed-in-Desktop

## QA test

1. Open Create in desktop mode
2. Close both the `Create Tools` and `Entity List` windows
3. Create should be de-activated.
4. Re-activate the Create app and repeat step 2 and 3.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12053/HighFidelity-Beta-PR13811-876079b.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12053/HighFidelity-Beta-PR13811-876079b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12053/HighFidelity-Beta-Interface-PR13811-876079b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12053/HighFidelity-Beta-PR13811-876079b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12053/HighFidelity-Beta-Interface-PR13811-876079b.exe

File: scripts/system/edit.js [hunk 1]
```diff
@@ -806,6 +806,14 @@ var toolBar = (function () {
 
         addButton("newMaterialButton", createNewEntityDialogButtonCallback("Material"));
 
+        var deactiveCreateIfDesktopWindowsHidden = function() {
```
Comment (huffman) [inline scripts/system/edit.js]: Can this be changed to be `deactivate...`?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12069/HighFidelity-Beta-PR13811-221dfb8.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12069/HighFidelity-Beta-PR13811-221dfb8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12069/HighFidelity-Beta-Interface-PR13811-221dfb8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12069/HighFidelity-Beta-PR13811-221dfb8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12069/HighFidelity-Beta-Interface-PR13811-221dfb8.exe

**Comment by qlimguicheteau** [comment]
PASS @conklin94122 

Attempted various patterns for closing the windows from using the x buttons to closing them on the taskbar, and intermingling the methods.

**Pull request merged**